packages/protocol/test/governance/vesting.ts

```diff
@@ -1069,10 +1022,10 @@ contract('Vesting', (accounts: string[]) => {
   describe('#revoke()', () => {
     it('revoker should be able to revoke the vesting', async () => {
       await createNewVestingInstanceTx(vestingDefaultSchedule, web3)
-      const vestingInstanceRegistryAddress = await vestingFactoryInstance.vestings(beneficiary)
-      const vestingInstance = await VestingInstance.at(vestingInstanceRegistryAddress)
-      const revokeBlockTimestamp = await getCurrentBlockchainTimestamp(web3)
+      const vestingInstanceAddress = await vestingFactoryInstance.vestings(beneficiary)
```

Comment:
ditto, pull into a before each?

Reply:
true, but there are tests in-between which change the behaviour of hte vesting scheme and construct a new vesting with that, hence I cannot set them the 3 lines in a common beforeEach